src/d3d9/d3d9_adapter.cpp

```diff
@@ -538,12 +538,19 @@ namespace dxvk {
     // Max Stream Stride
     pCaps->MaxStreamStride           = 508; // bytes
 
+
     const uint32_t majorVersion = options.shaderModel;
     const uint32_t minorVersion = options.shaderModel != 1 ? 0 : 4;
 
+    const uint32_t PSMajorVersion = options.PSShaderModel;
```

Comment:
To ease merging with the original dxvk, all changes to the original dxvk files need to be wrapped with comments, like:
```
// NV-DXVK start: DLFG integration
  code.goes.here();
// NV-DXVK end
```